### ISSUE TITLE
Impossible to force authentication for every use of a document.

### ISSUE THREAD

**Issue by pietrini** [eu-digital-identity-wallet/eudi-lib-android-wallet-core#108]
Hello everyone,

For documents today we have to provide a positive value for the
```kotlin
userAuthenticationTimeout = 1L
```

It would be great to have the possibility to provide an option to say I want the user to authenticate for every use of this document.

According to their code and implementation, it's a behavior supported by Android Identity SDK:
``` kotlin

/**
 * Class for holding Android Keystore-specific settings related to key creation.
 */
class AndroidKeystoreCreateKeySettings private constructor(

   ...

    /**
     * Gets whether user authentication is required.
     */
    val userAuthenticationRequired: Boolean,

    /**
     * The user authentication timeout, or 0 if authentication is required on every use.
     */
    val userAuthenticationTimeoutMillis: Long,

   ...
```

The only way to bypass this in the implementation is to call:
``` kotlin
    wallet.getDefaultCreateDocumentSettings {
        setUserAuthenticationRequired(
            required = true,
            timeoutMillis = 0,
            userAuthenticationTypes = setOf(
                UserAuthenticationType.LSKF, UserAuthenticationType.BIOMETRIC
            )
        )
    }
```

**Comment by vkanellopoulos** [comment]
Hi @pietrini 

Thank you for pointing this out! We’ll address this in the next release. We appreciate your feedback and patience as we work on the fix.

**Comment by pietrini** [comment]
My pleasure @vkanellopoulos !

**Comment by vkanellopoulos** [comment]
Fixed in release 0.12.2
